basedtyping/__init__.py

```diff
@@ -24,10 +24,20 @@
     Cannot actually be called unless it's narrowed, so it should only really be used as
     a bound in a ``TypeVar``.
     """
+
+    Never = NoReturn
 else:
     # for isinstance checks
     Function = Callable
 
+    class _NeverMetaclass(type):
+        def __instancecheck__(cls, instance: object) -> bool:
+            return False
+
+    class Never(metaclass=_NeverMetaclass):
+        pass
```

Comment:
Should this raise?

Reply:
when?

Reply:
In the metaclass `__call__`, and ban subtyping as well?